raft/src/main/java/org/apache/kafka/raft/QuorumStateStore.java

```diff
@@ -34,4 +43,10 @@
      * @throws IOException For any error encountered while writing the updated state
      */
     void write(ElectionState latest) throws IOException;
+
+    /**
+     * Cleanup quorum state store.
```

Comment:
I am not sure this needs to be in the interface. Outside of tests, do we need it?

Reply:
I think so, will remove it.